thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -154,6 +339,7 @@ export default Component.extend({
     const start = analysisRange[0] || '1548489600000';
     const end = analysisRange[1] || '1548748800000';
     const alertUrl = `yaml/preview?start=${start}&end=${end}&tuningStart=0&tuningEnd=0`;
```

Comment:
add '/' before 'yaml/preview' ?

Reply:
remove the hard coded timerange for start and end

Reply:
done and done